src/state/application/reducer.ts

```diff
@@ -31,6 +32,7 @@ export interface ApplicationState {
   readonly kncPrice?: string
   readonly chainIdWhenNotConnected: ChainId
   exchangeSubgraphClients: { [key: string]: ApolloClient<NormalizedCacheObject> }
+  readonly gasPrice?: { [key: string]: string }
```

Comment:
we can define more clearly type for gas price here. I think it should be: { standard: string, fast: string, low: string, default: string }

Reply:
agree